src/main/resources/hauth.yaml

```diff
@@ -170,6 +170,80 @@ paths:
                   message:
                     type: string
                     example: The access cookie decryption service is not configured properly
+  /token/sinai:
```

Comment:
Something to think about (I know you already raised this issue, just restating it here) -- is it possible to use OpenAPI and not bundle in specifics about individual services (since we're the only ones who will be using this, probably) ??

Reply:
Not sure if this would be a way to implement that, but [reference objects](https://swagger.io/specification/#reference-object) look pretty interesting

Reply:
Yes, I definitely think that would be a way to keep our definition cleaner and more concise. We're probably getting to the point with this file that refactoring using these would be beneficial. I'll create a ticket.